public/js/icinga/behavior/navigation.js

```diff
@@ -282,63 +318,102 @@
     };
 
     /**
-     * Show the fly-out menu
+     * Captures the mouse enter events to the navigation item and show the flyout.
      *
      * @param e
      */
-    Navigation.prototype.showFlyoutMenu = function(e) {
-        var $layout = $('#layout');
-
+    Navigation.prototype.onMouseEnter = function(e) {
+        let $layout = $('#layout');
+        let _this = e.data.self;
         if ($layout.hasClass('minimal-layout')) {
             return;
         }
 
-        var $target = $(this);
-        var $flyout = $target.find('.nav-level-2');
-
+        let $target = $(this);
+        let $flyout = $target.find('.nav-level-2');
         if (! $flyout.length) {
```

Comment:
This can be simplified to `! $target[0].matches(':has(.nav-level-2)')` which doesn't require a reference to the flyout itself.